examples/pitr/variables.tf

```diff
@@ -44,3 +44,10 @@ variable "pitr_id" {
   type        = string
   description = "The ID of the source deployment PostgreSQL instance that you want to recover back to. The PostgreSQL instance is expected to be in an up and in running state."
 }
+
+variable "members" {
+  type        = number
+  description = "Allocated number of members. Members can be scaled up but not down."
```

Comment:
I would mention here that the number of members has to match the number from the instance you are recovering from

Reply:
I am not sure if it has to match or just we cannot scale down it.

Reply:
ah ok, thats probably it. Update the description to say it must be the same or higher